config/features.yml

```diff
@@ -57,6 +57,10 @@ features:
     actor_type: user
     enable_in_development: true
     description: A fast and dirty way to get the operating status from lighthouse
+  facility_locator_lighthouse_covid_vaccine_query:
```

Comment:
looks like FE is looking for `facilityLighthouseCovidVaccineQuery` which doesn't match with this, does it?

Reply:
This should fix it - https://github.com/department-of-veterans-affairs/vets-website/pull/15502/commits/3cfc68017ce90b0e330dfdc6ddc57749753d3152

Reply:
Thanks for the update!